person.rb

```diff
@@ -9,6 +10,7 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @age = age
     @name = name
     @parent_permission = parent_permission
+    @rentals = []
```

Comment:
- [ ] Please also add here a method to add rentals using the Person class. The person himself should be sent as a parameter to create the new rental achieving the 'has-many' association.